tests/dhcp/test_dnsmasq.c

```diff
@@ -36,6 +40,17 @@ static char *test_dhcp_conf_content =
     "dhcp-range=wifi_if.2,10.0.2.2,10.0.2.254,255.255.255.0,24h\n"
     "dhcp-range=wifi_if.3,10.0.3.2,10.0.3.254,255.255.255.0,24h\n";
 
+static char *test_dhcp_conf_prefix_if =
+    "no-resolv\n"
+    "server=8.8.4.4\n"
+    "server=8.8.8.8\n"
+    "dhcp-leasefile=/tmp/test_dnsmasq.leases\n"
+    "dhcp-script=/tmp/dnsmasq_exec-test.sh\n"
+    "dhcp-range=eth_if0,10.0.0.2,10.0.0.254,255.255.255.0,24h\n"
+    "dhcp-range=eth_if1,10.0.1.2,10.0.1.254,255.255.255.0,24h\n"
+    "dhcp-range=eth_if2,10.0.2.2,10.0.2.254,255.255.255.0,24h\n"
+    "dhcp-range=eth_if3,10.0.3.2,10.0.3.254,255.255.255.0,24h\n";
+
 // why aren't we using amazing C++11 which has the R"(...) string literal??? 😭
```

Comment:
I was literally just about to comment this same thing :sob:

Reply:
Rust :)